pkg/sqlreplay/replay/replay_test.go

```diff
@@ -222,6 +224,7 @@ func TestProgress(t *testing.T) {
 			// Maybe unstable due to goroutine schedule.
 			// require.LessOrEqual(t, progress, float64(i+2)/10)
 		}
+		replay.Wait()
```

Comment:
A tiny fix. The test `TestProgress` is not stable because the next `Start` may happen before it finished.